andela/ah-frontend-tesseract
# #161210031 users should get error message for existing username or password

#### What does this PR do?
fixes the bug where signup was not showing any message on attempt to signup with existing username or email.
#### Description of Task to be completed?
- already taken username or email message display
- change social signup buttons to all show login
- removed twitter button
#### How should this be manually tested?
- navigate to register, 
- attempt to create a user using an existing username or password
- appropriate error messages should be shown.
### What are the relevant pivotal tracker stories?
[#161210031](https://www.pivotaltracker.com/story/show/161210031)
#### Screenshots (if appropriate)
<img width="952" alt="screen shot 2018-10-15 at 14 48 42" src="https://user-images.githubusercontent.com/30045448/46949286-9002ce00-d089-11e8-8b8c-7a41a83be4b7.png">
